### ISSUE TITLE
The search stopped working

### ISSUE THREAD

**Issue by pustovalov** [Microsoft/vscode#59319]
Issue Type: <b>Bug</b>

Steps to Reproduce:

1. Try to search some text in files
2. See endless search

Video: https://monosnap.com/file/SS3Cvu2l6mJLUb9qVUNu8FbUSVQmtt

VS Code version: Code - Insiders 1.28.0-insider (fe8d9d2470ddffb9dc2da604b23f5bd8a185cd66, 2018-09-25T07:29:33.175Z)
OS version: Darwin x64 18.0.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6660U CPU @ 2.40GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 5, 8|
|Memory (System)|16.00GB (0.20GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (37)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|9.0.0
solargraph|cas|0.17.6
path-intellisense|chr|1.4.2
bracket-pair-colorizer|Coe|1.0.60
cjsx|cri|0.0.6
vscode-svgviewer|css|1.4.6
vscode-eslint|dba|1.6.0
vscode-faker|dee|1.3.0
githistory|don|0.4.2
remotehub|eam|0.2.0
EditorConfig|Edi|0.12.4
prettier-vscode|esb|1.6.1
mithril-emmet|Fal|0.7.1
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
vscode-flow-ide|gca|1.2.0
vscode-pull-request-github|Git|0.1.6
gc-excelviewer|Gra|2.1.26
haml|kar|1.1.0
ruby-syntax-replacer|Kos|1.0.11
atom-keybindings|ms-|3.0.4
debugger-for-chrome|msj|4.10.1
vscode-docker|Pet|0.3.0
material-icon-theme|PKi|3.6.0
vscode-css-peek|pra|2.1.0
ruby|reb|0.20.0
vscode-icons|rob|7.27.0
sass-indented|rob|1.4.9
code-settings-sync|Sha|3.1.2
code-spell-checker|str|1.6.10
lodash-snippets|tom|1.0.0
gitblame|wad|2.4.4
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.7.0
ReactSnippets|xab|2.2.0
html-css-class-completion|Zig|1.17.1
vscode-open-in-github|ziy|1.3.5

(3 theme extensions excluded)

</details>

**Comment by bpasero** [comment]
@pustovalov is it possible that you have the keybindings editor open?

**Comment by pustovalov** [comment]
@bpasero I turned off this extension: https://marketplace.visualstudio.com/items?itemName=Zignd.html-css-class-completion, and the search began to work

**Comment by zackkrida** [comment]
Same issue, no keybindings editor open but using the same class-completion extension as @pustovalov 

Disabling the plugin resolved the search issue.

**Comment by donovanhiland** [comment]
Odd. Experiencing the same issue. Disabling the plugin worked.

**Comment by payonel** [comment]
It might be best for me to create a new ticket, I hope my comment gets noticed.
I also hit this issue and nothing was helping with regards to extensions. I also experienced the endless search when trying to open/edit/run a task

Solution:
```
// ."git.enabled": false
```

I disable git integration in my vsode settings, and by simple commenting out that one line in my user settings (all extensions enabled, all settings still in use) now search is working again

**Comment by angellindo** [comment]
samething as the video, I disabled all the extensions and still the same

**Comment by roblourens** [comment]
That extension activates on `*` and blocks its activation on a findFiles. Then search waits on `*` activations...

**Comment by sandy081** [comment]
Does this need a special repo to verify it. I tested on my vscode workspace with a 7 days old build and with the mentioned extension and I do not see this issue.

**Comment by martellaj** [comment]
@sandy081 That might be too old. I started noticing it 2 days ago on that day's most recent Insiders build.

**Comment by roblourens** [comment]
It was only broken between I think last Friday and this Tuesday.

**Comment by payonel** [comment]
it also stopped working for me with ONLY `git.enabled = false` in my settings
and with this build it is fixed
